app/controllers/issue_templates_controller.rb

```diff
@@ -88,17 +94,22 @@ def set_pulldown
     grouped_options = []
     group = []
     default_template = nil
-    setting = IssueTemplateSetting.find_or_create(@project.id)
+    project_id = @project.id
+    tracker_id = @tracker.id
+    setting = IssueTemplateSetting.find_or_create(project_id)
     inherit_template = setting.enabled_inherit_templates?
 
-    project_ids = inherit_template ? @project.ancestors.collect(&:id) : [@project.id]
-    issue_templates = IssueTemplate.search_by_project(@project.id)
-                                   .search_by_tracker(@tracker.id)
+    project_ids = inherit_template ? @project.ancestors.collect(&:id) : [project_id]
+    issue_templates = IssueTemplate.search_by_project(project_id)
+                                   .search_by_tracker(tracker_id)
                                    .enabled.order_by_position
 
     project_default_template = issue_templates.is_default.first
 
-    unless project_default_template.blank?
+    has_project_default_template =  project_default_template.present?
```

Comment:
__[rubocop]__
* Operator `=` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Unnecessary spacing detected.